modules/utxo-lib/src/networks.ts

```diff
@@ -116,6 +117,15 @@ export const networks: Record<NetworkName, Network> = {
     wif: 0xef,
     coin: coins.BTC,
   },
+  bitcoinPublicSignet: {
+    messagePrefix: '\x18Bitcoin Signed Message:\n',
+    bech32: 'tb',
+    bip32: getDefaultBip32Testnet(),
+    pubKeyHash: 0x6f,
+    scriptHash: 0xc4,
+    wif: 0xef,
+    coin: coins.BTC,
```

Comment:
please add source for the constants as a review comment

Reply:
don't have a source, but I believe the public signet constants are the same as testnet - please correct me if I am wrong

Reply:
nvm, I think I do need to change some of the constants - esp. the header (`0xecc7daa2`)

Reply:
maybe not in this specific code block though